ticdc/ticdc-faq.md

```diff
@@ -188,8 +188,9 @@ TiCDC 使用 PD 内部的 etcd 来存储元数据并定期更新。因为 etcd 
 
 ## TiCDC 支持同步大事务吗？有什么风险吗？
 
-TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同可能存在以下风险：
+根据场景不同，TiCDC 同步大事务存在以下风险：
 
++ 可能导致主从同步延迟大幅增高
 + 当 TiCDC 内部处理能力不足时，可能出现同步任务报错 `ErrBufferReachLimit`。
 + 当 TiCDC 内部处理能力不足或 TiCDC 下游吞吐能力不足时，可能出现内存溢出 (OOM)。
 
```

Comment:
if errors about big transaction  happen,  enabling the splitting big transaction is perhaps a better choice